autotst/reaction.py

```diff
@@ -71,10 +71,66 @@
 
 class Reaction():
 
-    rmg_database = None   # will be an RMGDatabase instance, once loaded.
     # a dictionary will have reaction family names as keys and
     # TransitionStates instances as values, once loaded.
+    global ts_databases
```

Comment:
I'm a bit confused by the choice to load the databases every time a reaction object is initiated. Wasn't that why we moved to a `load_databases` method that would only load them if need be? It looks like this is all a copy of `load_databases` anyway... Could you elaborate on why this is here?